core/controllers/topics_and_skills_dashboard.py

```diff
@@ -76,11 +76,46 @@ def get(self):
 
         untriaged_skill_summary_dicts = []
         mergeable_skill_summary_dicts = []
+        categorized_skills_dict = {}
+        topics = topic_fetchers.get_all_topics()
+        for topic in topics:
+            topic = topic.to_dict()
+            subtopics = topic['subtopics']
+            categorized_skills_dict[topic['name']] = {}
```

Comment:
Note that topic names can collide with 'untriaged_skills' key.

Reply:
done